### DISCUSSION TITLE
Broken link in WebMIDI doc

### DISCUSSION THREAD

**Discussion by Stokestack** [mdn/mdn-community#152] [Content] [answered]
I'm not sure if this is the right place to report this, but there's a "supporting browsers" link in [this article](https://developer.mozilla.org/en-US/docs/Web/API/Web_MIDI_API#browser_compatibility) that goes nowhere.

**Comment by teoli2003** [comment, marked as answer]
Hi! 

Can you please open an issue by following this [link](https://github.com/mdn/content/issues/new?assignees=&labels=needs+triage&template=content-bug.yml)? 

That way, the problem will be tracked (discussions are not suited for tracking issues).

Thank you very much!

**Comment by Stokestack** [comment]
[Done](https://github.com/mdn/content/issues/18366).

**Reply by teoli2003** [reply]
Thank you!
